tartufo/scanner.py

```diff
@@ -691,7 +690,6 @@ def load_repo(self, repo_path: str) -> pygit2.Repository:
         except (FileNotFoundError, types.ConfigException):
             config_file = None
         if config_file and config_file != self.global_options.config:
-            self.config_data = data
```

Comment:
We need to be certain that we still load excluded signatures from the target repository's config somehow.